test/cdk-lib/bedrock/agent-action-group.test.ts

```diff
@@ -47,11 +46,12 @@ describe('AgentActionGroup', () => {
       },
     });
 
-    agent = new bedrock.Agent(stack, 'Agent', {
-      foundationModel: bedrock.BedrockFoundationModel.ANTHROPIC_CLAUDE_V2_1,
-      instruction: 'You provide support for developers working with CDK constructs.',
-      idleSessionTTL: cdk.Duration.minutes(30),
-    });
+    // agent = new bedrock.Agent(stack, 'Agent', {
```

Comment:
couple of commented tests in this file

Reply:
updated - a5b222881d2860c503ed38fcd68ea6619e07542a